package-lock.json

```diff
@@ -0,0 +1,214 @@
+{
```

Comment:
Why do we need a `package-lock.json` on the root?

Reply:
Oh I will remove that Asap